recipe/meta.yaml

```diff
@@ -25,6 +27,8 @@ requirements:
     - _pytorch_select  {{ pytorch_select_version }}
     - pytorch-base     {{ pytorch }}
     - setuptools {{ setuptools }}
+    - ffmpeg {{ ffmpeg }}
+    - av
```

Comment:
Should `av` have a version tag as well?

Reply:
As of now  `av` is just required by `torchvision`, so have not added a tag yet. Please suggest.

Reply:
I think originally we'd intended to favor being more strict (greater versioning enforcement), for greater certainty.  It may not matter right now for this one, with only one dependent, but we'd need to remember to add versioning later if other dependents came along, and it would be easier to just do it strictly up front -- at least, for those packages that are coming from within the `open-ce` project itself, like `av`.

But as a broader comment, we're not being as consistent with some native build tools used from outside `open-ce`.  For example, three different feedstocks take three different approaches for something like `libtool`:
```
jpeg-turbo-feedstock/recipe/meta.yaml:    - libtool {{ libtool }}
libevent-feedstock/recipe/meta.yaml:    - libtool
libflac-feedstock/recipe/meta.yaml:    - libtool  # [unix]
```
...that's just an example, beyond the scope of this `av` PR, but since `av` is part of `open-ce` universe I would think we should go ahead and version it; my opinion.

Reply:
Sure, so I will add `av` tag as well.

Reply:
It looks like pyav uses semantic versioning (https://semver.org/) so we should be able add the pin at `8.*`